jsk_pcl_ros_utils/sample/sample_add_point_indices.launch

```diff
@@ -0,0 +1,106 @@
+<launch>
+
+  <arg name="gui" default="true" />
+
+  <param name="/use_sim_time" value="true" />
+  <rosparam command="load"
+            file="$(find jsk_pcl_ros)/test_data/2015-11-04-19-37-29_baxter-kiva-object-in-hand-cloud/rosparam.yaml" />
```

Comment:
This is **test_data** and it is not installed when people runs `catkin build`.
Please move this rosbag downlod from install_test_data.py to install_sample_data.py, and runs it in CMakeLists.txt.
For example:
 - https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_perception/scripts/install_sample_data.py
 - https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_perception/CMakeLists.txt#L75